db/seeds.rb

```diff
@@ -5,70 +5,95 @@
 require 'paper_trail'
 require 'timecop'
 
-include FactoryBot::Syntax::Methods
+module Seeds
```

Comment:
Why did you put everything in a module?

Reply:
Because you are supposed to include/extend inside a module, not at top level.

Reply:
Huh.

Reply:
At least that's what rubocop says.

Reply:
It just seems silly because we never use this module anywhere. Instead of `include`ing factorybot methods you could require the file and then call `FactoryBot.create` instead of just `create` in space. _that_ is truly the proper way.